submissions/2020-07-27-OpenFF-Benchmark-Ligands/compute.json

```diff
@@ -0,0 +1,63 @@
+{
+  "qc_specifications": {
+    "ani2x": {
+      "method": "ani2x",
+      "basis": null,
+      "program": "torchani",
+      "spec_name": "ani2x",
+      "spec_description": "ANI2x ML potential.",
+      "store_wavefunction": "none"
+    }
+  },
+  "dataset_name": "OpenFF-benchmark-ligand-fragments-v1.0",
+  "dataset_tagline": "OpenForcefield TorsionDrives.",
+  "dataset_type": "TorsiondriveDataset",
+  "maxiter": 200,
+  "driver": "gradient",
+  "scf_properties": [
+    "dipole",
+    "quadrupole",
+    "wiberg_lowdin_indices",
+    "mayer_indices"
+  ],
+  "priority": "normal",
+  "description": "A TorsionDrive dataset using geometric.",
```

Comment:
Same as above question. I anticipate this doesn't matter, but thought it worth asking. I don't consider it a blocker for submission today.